modules/interactive_music/editor/audio_stream_interactive_editor_plugin.cpp

```diff
@@ -187,54 +182,56 @@ void AudioStreamInteractiveTransitionEditor::_update_transitions() {
 			if (!exists) {
 				if (audio_stream_interactive->has_transition(AudioStreamInteractive::CLIP_ANY, to)) {
 					from = AudioStreamInteractive::CLIP_ANY;
-					tooltip = "Using Any Clip -> " + audio_stream_interactive->get_clip_name(to) + ".";
+					tooltip = vformat(TTR("Using Any Clip -> %s."), audio_stream_interactive->get_clip_name(to));
 				} else if (audio_stream_interactive->has_transition(from, AudioStreamInteractive::CLIP_ANY)) {
 					to = AudioStreamInteractive::CLIP_ANY;
-					tooltip = "Using " + audio_stream_interactive->get_clip_name(from) + " -> Any Clip.";
+					tooltip = vformat(TTR("Using %s -> Any Clip."), audio_stream_interactive->get_clip_name(from));
 				} else if (audio_stream_interactive->has_transition(AudioStreamInteractive::CLIP_ANY, AudioStreamInteractive::CLIP_ANY)) {
 					from = to = AudioStreamInteractive::CLIP_ANY;
-					tooltip = "Using All CLips -> Any Clip.";
+					tooltip = TTR("Using All Clips -> Any Clip.");
 				} else {
-					tooltip = "No transition available.";
+					tooltip = TTR("No transition available.");
 				}
 			}
 
+			String from_time;
+			String to_time;
 			if (audio_stream_interactive->has_transition(from, to)) {
 				icon = fade_icons[audio_stream_interactive->get_transition_fade_mode(from, to)];
 				switch (audio_stream_interactive->get_transition_from_time(from, to)) {
 					case AudioStreamInteractive::TRANSITION_FROM_TIME_IMMEDIATE: {
-						txt += TTR("Immediate");
+						from_time = TTR("Immediate");
 					} break;
 					case AudioStreamInteractive::TRANSITION_FROM_TIME_NEXT_BEAT: {
-						txt += TTR("Next Beat");
+						from_time = TTR("Next Beat");
 					} break;
 					case AudioStreamInteractive::TRANSITION_FROM_TIME_NEXT_BAR: {
-						txt += TTR("Next Bar");
+						from_time = TTR("Next Bar");
 					} break;
 					case AudioStreamInteractive::TRANSITION_FROM_TIME_END: {
-						txt += TTR("Clip End");
+						from_time = TTR("Clip End");
 					} break;
 					default: {
 					}
 				}
 
 				switch (audio_stream_interactive->get_transition_to_time(from, to)) {
 					case AudioStreamInteractive::TRANSITION_TO_TIME_SAME_POSITION: {
-						txt += TTR(L"⮕ Same");
+						to_time = TTR("Same", "Transition Time Position");
 					} break;
 					case AudioStreamInteractive::TRANSITION_TO_TIME_START: {
-						txt += TTR(L"⮕ Start");
+						to_time = TTR("Start", "Transition Time Position");
 					} break;
 					case AudioStreamInteractive::TRANSITION_TO_TIME_PREVIOUS_POSITION: {
-						txt += TTR(L"⮕ Prev");
+						to_time = TTR("Prev", "Transition Time Position");
 					} break;
 					default: {
 					}
 				}
 			}
 
 			rows[j]->set_icon(i, icon);
-			rows[j]->set_text(i, txt);
+			rows[j]->set_text(i, to_time.is_empty() ? from_time : vformat(U"%s⮕%s", from_time, to_time));
```

Comment:
Whitespace lost in translation:
```suggestion
			rows[j]->set_text(i, to_time.is_empty() ? from_time : vformat(U"%s ⮕ %s", from_time, to_time));
```